tests/test_helpers.py

```diff
@@ -57,5 +59,104 @@ def test_extract_keyword(self):
             keyword_got = extract_keyword(keyword_line)
 
 
+    cross_references_testdata = [
+        # Testing for normal input.
+        [
+            "<xref uid=\"google.cloud.bigquery_storage_v1.types.SplitReadStreamResponse\">google.cloud.bigquery_storage_v1.types.SplitReadStreamResponse</xref>",
+            "google.cloud.bigquery_storage_v1.types.SplitReadStreamResponse"
+        ],
+        # Testing for no cross references to convert.
+        [
+            "Response message for SplitReadStreamResponse.",
+            "Response message for SplitReadStreamResponse."
+        ],
+        # Testing for cross references to convert within longer content.
+        [
+            "Response message for <xref uid=\"google.cloud.bigquery_storage_v1.types.SplitReadStreamResponse\">google.cloud.bigquery_storage_v1.types.SplitReadStreamResponse</xref>.",
+            "Response message for google.cloud.bigquery_storage_v1.types.SplitReadStreamResponse."
+        ],
+    ]
+    @parameterized.expand(cross_references_testdata)
+    def test_convert_cross_references(self, content_want, content):
```

Comment:
Nit: would you mind swapping the order of content and want? That way, when you're reading the test cases from top to bottom, you start with the input then see the expected output. On first read, I was confused why the `xref` was getting removed.

Reply:
Agreed. Done.